docs/core/whats-new/whats-new-in-dotnet-core-2-0.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What's new in .NET Core 2.0
+description: A description of the new feature found in .NET Core 2.0.
```

Comment:
A description should be a call to action. So I suggest something along the lines of "Learn about the new features found in .NET Core"